package.json

```diff
@@ -39,8 +39,10 @@
     "moment-duration-format": "^1.3.0",
     "moment-parser": "^0.3.0",
     "moment-timezone": "^0.4.0",
+    "node-grok": "github:beh01der/node-grok#b76dae6dc4efa745e6bf75a3cb8d113aa47bc16e",
```

Comment:
They haven't published the 1.0.5 version yet and it has the recent and working copy of oniguruma which actually builds fine locally for me and on travis (unlike the one in 1.0.4 which is broken, ie oniguruma@4.2.1 ), here's the specific commit: 

https://github.com/Beh01der/node-grok/commit/bf961bc3b8301f491314a8af62fa69f3927bec48

I could poke at the owner to publish the latest 1.0.5 build.